python/langsmith/client.py

```diff
@@ -4664,6 +4664,538 @@ def _evaluate_strings(
             **kwargs,
         )
 
+    def _get_settings(self) -> dict:
+        """Get the settings for the current tenant.
+
+        Returns:
+            dict: The settings for the current tenant.
+        """
+        response = self.request_with_retries("GET", "/settings")
```

Comment:
Would be good to cache this so we don't have to send 2 api requests on every pull